wgpu-native/src/hub.rs

```diff
@@ -87,107 +105,109 @@ impl IdentityManager {
     }
 }
 
-pub struct Storage<T> {
+pub struct Storage<T, I:'static + TypedId> {
```

Comment:
Is the `'static` bound needed for these generics? It seems like we could just have `PhantomData<I>`

Reply:
Yes you are right, I changed it.